aboutcitus/concepts.rst

```diff
@@ -0,0 +1,125 @@
+.. _citus_concepts:
+
+
+Concepts
+========
+
+This section is a brief introduction to common terms in our documentation, with links to more complete information.
+
+Distributed Architecture
+------------------------
+
+Citus is a PostgreSQL `extension <https://www.postgresql.org/docs/9.6/static/external-extensions.html>`_ that allows commodity database servers (called *nodes*) to coordinate with one another in a "shared nothing" architecture. The nodes form a *cluster* that allows PostgreSQL to hold more data and use more CPU cores than would be possible on a single computer. This architecture also allows the database to scale by simply adding more nodes to the cluster.
+
+Nodes: Coordinator and Workers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Every cluster has one special node called the *coordinator* (the others are known as workers). Applications send their queries to the coordinator node which relays it to the relevant workers and accumulates the results.
+
+For each query, the coordinator either *routes* it to a single worker node, or *parallelizes* it across several depending on whether the required data lives on a single node or multiple.  The coordinator knows how to do this by consulting its metadata tables. These Citus-specific tables track the DNS names and health of worker nodes, and the distribution of data across nodes. For more information, see our :ref:`metadata_tables`.
+
+Citus also supports an :ref:`mx` mode to allow queries directly against workers. This allows both reads and writes to be scaled even more across all nodes.
+
+Table Types
+-----------
+
+There are three types of tables in a Citus cluster, each used for different purposes.
+
+Type 1: Distributed Tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The first type, and most common, is *distributed* tables. These appear to be normal tables to SQL statements, but are horizontally *partitioned* across worker nodes.
+
+.. image:: ../images/diagram-parallel-select.png
+
+Here the rows of ``table`` are stored in tables ``table_1001``, ``table_1002`` etc on the workers. The component worker tables are called *shards*.
+
+Citus runs not only SQL but DDL statements throughout a cluster, so changing the schema of a distributed table cascades to update all the table's shards across workers. See :ref:`ddl`.
```

Comment:
Can we link to our DDL section on Creating Distributed Tables? (e.g.: You can learn how to create distributed tables <here>" or some such.